src/core/proto_reader/reader.rs

```diff
@@ -178,21 +178,19 @@ mod test_proto_config {
         let file_rt = runtime.file.clone();
 
         let reader = ProtoReader::init(ResourceReader::<Cached>::cached(runtime.clone()), runtime);
-        let helper_map = reader
+        let file_descriptors = reader
             .resolve(reader.read_proto(&test_file, None).await?, Some(test_dir))
             .await?;
-        let files = test_dir.read_dir()?;
-        for file in files {
-            let path = file?.path();
+        for file in file_descriptors
+            .iter()
+            .filter(|desc| !desc.name.as_ref().unwrap().starts_with("google/protobuf/"))
```

Comment:
```suggestion
            .filter_map(|desc| !desc.name.as_ref().map(|a| a.starts_with("google/protobuf/")))
```

Reply:
it's a code in the test I think it's okay to leave the unwrap. I expect it to fail here in such a case instead of ignoring it. And that adds more logic in the test itself though it's already has too much conditions that makes it less declarative and understandable for a test